### PR TITLE
/transports/create endpoint

### PR DESCRIPTION
**Pull request by sagakortesaari**
We now have an endpoint for transports. Currently supports creation of new transports.

### CODE REVIEW

**Comment by sagakortesaari** [comment]
Added endpoint /transports/search/id aswell - we can now search transports by ID's

File: flaskr/transports.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+$ curl -X POST -d '{ "_id": [1,2] }' -H "Content-Type: application/json" 127.0.0.1:5000/transports/search/id
+"""
+
+@app.route('/transports/search/id', methods=['POST'])
+def transports_get_id():
+    query = request.get_json()
+    succ, msg = check_payload(transport_search_sample_id, query)
+
+    if succ:
+        found = coll_transports.find({'_id': {'$in': query['_id']}}) # find products with any of the id's provided
```
Comment (dcronqvist) [inline flaskr/transports.py]: Should it not be transports in the comment?

**Review by dcronqvist — CHANGES_REQUESTED** [review]
i left a single line comment about a comment

File: flaskr/transports.py (hunk 1 above)
Comment (sagakortesaari) [reply flaskr/transports.py]: It should! Fixing ASAP

**Review by dcronqvist — APPROVED** [review]
This looks really good

**Pull request merged**